docs/pt/docs/python-types.md

```diff
@@ -0,0 +1,316 @@
+
+Mas, infelizmente, você não obtém nada útil:
+
+<img src="/img/python-types/image01.png">
+
+### Adicionar tipos
+
+Vamos modificar uma única linha da versão anterior.
+
+Vamos mudar exatamente esse fragmento, os parâmetros da função, de:
+
+```Python
+    first_name, last_name
+```
+
+para:
+
+```Python
+    first_name: str, last_name: str
+```
+
+É isso aí.
+
+Esses são os "type hints":
+
+```Python hl_lines="1"
+{!../../../docs_src/python_types/tutorial002.py!}
+```
+
+Isso não é o mesmo que declarar valores padrão como seria com:
+
+```Python
+    first_name="john", last_name="doe"
+```
+
+É uma coisa diferente.
+
+Estamos usando dois pontos (`:`), não é igual a (`=`).
+
+E adicionar type hints normalmente não muda o que acontece do que aconteceria sem elas.
+
+Mas agora, imagine que você está novamente no meio da criação dessa função, mas com type hints.
+
+No mesmo ponto, você tenta acionar o preenchimento automático com o `Ctrl Space` e vê:
+
+<img src="/img/python-types/image02.png">
+
+Com isso, você pode rolar, vendo as opções, até encontrar o que "toca uma campainha":
+
+<img src="/img/python-types/image03.png">
+
+## Mais motivação
+
+Marque esta função, ela já possui type hints:
+
+```Python hl_lines="1"
+{!../../../docs_src/python_types/tutorial003.py!}
+```
+
+Como o editor conhece os tipos de variáveis, você não apenas obtém a conclusão, mas também as verificações de erro:
+
+<img src="/img/python-types/image04.png">
+
+Agora você sabe que precisa corrigí-lo, converta `age` em uma string com `str (age)`:
+
+```Python hl_lines="2"
+{!../../../docs_src/python_types/tutorial004.py!}
+```
+
+## Tipos de declaração
+
+Você acabou de ver o local principal para declarar type hints. Como parâmetros de função.
+
+Este também é o principal local em que você os usaria com o **FastAPI**.
+
+### Tipos simples
+
+Você pode declarar todos os tipos padrão de Python, não apenas `str`.
+
+Você pode usar, por exemplo:
+
+* `int`
+* `float`
+* `bool`
+* `bytes`
+
+```Python hl_lines="1"
+{!../../../docs_src/python_types/tutorial005.py!}
+```
+
+### Tipos genéricos com parâmetros de tipo
+
+Existem algumas estruturas de dados que podem conter outros valores, como `dict`, `list`, `set` e `tuple`. E os valores internos também podem ter seu próprio tipo.
+
+Para declarar esses tipos e os tipos internos, você pode usar o módulo Python padrão `typing`.
+
+Ele existe especificamente para suportar esses type hints.
+
+#### `List`
+
+Por exemplo, vamos definir uma variável para ser uma `lista` de `str`.
+
+Em `typing`, importe `List` (com um `L` maiúsculo):
+
+```Python hl_lines="1"
+{!../../../docs_src/python_types/tutorial006.py!}
+```
+
+Declare a variável com a mesma sintaxe de dois pontos (`:`).
+
+Como o tipo, coloque a `List`.
+
+Como a lista é um tipo que contém alguns tipos internos, você os coloca entre colchetes:
+
+```Python hl_lines="4"
+{!../../../docs_src/python_types/tutorial006.py!}
+```
+
+!!! dica
```

Comment:
I think you need to keep the `tip` word to mark this as a tip box like this (same thing with info in other places):
![image](https://user-images.githubusercontent.com/20428941/89741409-76edbd00-da67-11ea-9f0f-6286c0134a1b.png)
![image](https://user-images.githubusercontent.com/20428941/89741423-91279b00-da67-11ea-890b-104bb134eae5.png)


```suggestion
!!! tip "Dica"
```

Reply:
Well observed, I will change. Thank you ☺️